setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
+#ignore line limit error
+ignore = E501,E302,E261,E262,E701,E241,E126,E127,E128,E225
```

Comment:
We don't want to ignore all these errors, but only the line limit error for now.

Reply:
okay